core/primitives/src/views.rs

```diff
@@ -345,10 +345,33 @@ pub struct DebugBlockStatus {
     pub timestamp_delta: u64,
 }
 
+#[cfg_attr(feature = "deepsize_feature", derive(deepsize::DeepSizeOf))]
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq)]
+pub struct PeerInfoView {
+    pub addr: String,
+    pub account_id: Option<AccountId>,
+    pub height: BlockHeight,
+    pub tracked_shards: Vec<ShardId>,
+    pub archival: bool,
+}
+
+#[cfg_attr(feature = "deepsize_feature", derive(deepsize::DeepSizeOf))]
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq)]
+pub struct NetworkInfoView {
+    pub peer_max_count: u32,
+    pub num_connected_peers: usize,
+    pub connected_peers: Vec<PeerInfoView>,
+    pub sent_bytes_per_sec: u64,
+    pub received_bytes_per_sec: u64,
+    pub peer_counter: usize,
```

Comment:
what is 'peer_counter'?

Reply:
I think rate-limiting is taken into account.

Reply:
See https://github.com/near/nearcore/blob/c43516499a972b066c0e6f649c88597192f2c7e8/chain/network/src/peer_manager/peer_manager_actor.rs#L188

Reply:
Frol has a point. Let's clean up the fields in this struct to remove the parts we are not displaying. We can always add more later if we find anything useful.